ang/civicase/Utils.js

```diff
@@ -83,27 +84,35 @@
       act.is_overdue = (typeof act.is_overdue === 'string') ? (act.is_overdue === '1') : act.is_overdue;
       act.color = activityStatuses[act.status_id].color || '#42afcb';
       act.status_css = 'status-type-' + act.status_type + ' activity-status-' + act.status_name.toLowerCase().replace(' ', '-');
+
+      formatActivityContactNames(act);
```

Comment:
Hello! I removed this function because it was not necessary. 

The idea behind it was to add the contact name to the activity if it only had the contact id. I didn't know that you could pass `assignee_contact_name` to the API since it didn't appear as an option in the API explorer, but I investigated a bit and it is a valid parameter that returns the information needed.

BTW, this information is used to display the contact names in the activity card.